js/classes/entities/midground/enemies/slime.js

```diff
@@ -142,6 +142,11 @@ class Slime {
             }
         } else if (deathAnim.currentFrame() === deathAnim.frameCount - 1) {
             this.removeFromWorld = true;
+            if (STORY.slimesKilled) {
```

Comment:
I'm a little confused on what this means. If any slimes are killed increment otherwise set equal to 1? Why are we setting equal to 1? Also feels weird that we ware using the same variable for boolean as for incrementing.

Reply:
In JS, 0 is falsy, so (!STORY.slimesKilled) is the same thing as (STORY.slimesKilled === 0). However, that's not exactly the case here. The counter is undefined, it's never set to zero. So, if it's undefined, set it to 1. If it's defined, increment